app/code/Magento/Customer/Model/Customer.php

```diff
@@ -485,8 +491,9 @@ public function getAddressById($addressId)
     /**
      * Getting customer address object from collection by identifier
      *
-     * @param int $addressId
+     * @param $addressId
```

Comment:
Looks like you removed type of this parameter by mistake, please revert this change

Reply:
**done**